nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/cluster/ui/common/cluster-table-filter/cluster-table-filter.component.html

```diff
@@ -38,5 +35,6 @@
                 </mat-form-field>
             </div>
         </div>
+        <div class="py-2 accent-color font-medium">Filter matched {{ filteredCount }} of {{ totalCount }}</div>
```

Comment:
The spacing here is off compared with the other pages.

![Screenshot 2024-05-17 at 4 03 53 PM](https://github.com/apache/nifi/assets/123395/1e3af880-44c7-49f6-a377-257cd294c8ca)

Reply:
Good catch!